src/azure-cli/azure/cli/command_modules/appconfig/_help.py

```diff
@@ -147,8 +147,8 @@
     text: az appconfig kv list --key color --connection-string Endpoint=https://contoso.azconfig.io;Id=xxx;Secret=xxx --label v1.*
   - name: List all keys with any labels and query only key, value and tags.
     text: az appconfig kv list --connection-string Endpoint=https://contoso.azconfig.io;Id=xxx;Secret=xxx --fields key value tags --datetime "2019-05-01T11:24:12Z"
-  - name: List 150 key-values with any labels.
-    text: az appconfig kv list --connection-string Endpoint=https://contoso.azconfig.io;Id=xxx;Secret=xxx  --top 150
+  - name: List content of key vault reference with key prefix 'KVRef_'.
+    text: az appconfig kv list --connection-string Endpoint=https://contoso.azconfig.io;Id=xxx;Secret=xxx  --key "KVRef_" --resolve-keyvault
```

Comment:
>"KVRef_" [](start = 113, length = 8)

--key KVRef_*

Reply:
Will fix.

---
In reply to: [405713778](https://github.com/Azure/azure-cli/pull/12893#discussion_r405713778) [](ancestors = 405713778)